OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ApplicationController.cs

```diff
@@ -43,30 +43,30 @@ public class ApplicationController : ControllerBase
         this.userService = userService;
     }
 
-    /// <summary>
-    /// Get all applications from the database.
-    /// </summary>
-    /// <param name="filter">Application filter.</param>
-    /// <returns>List of all applications.</returns>
-    /// <response code="200">All entities were found.</response>
-    /// <response code="204">No entity was found.</response>
-    /// <response code="500">If any server error occurs.</response>
-    [HasPermission(Permissions.ApplicationRead)]
-    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(SearchResult<ApplicationDto>))]
-    [ProducesResponseType(StatusCodes.Status204NoContent)]
-    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-    [HttpGet]
-    public async Task<IActionResult> Get([FromQuery] ApplicationFilter filter)
-    {
-        var applications = await applicationService.GetAll(filter).ConfigureAwait(false);
-
-        if (!applications.Entities.Any())
-        {
-            return NoContent();
-        }
-
-        return Ok(applications);
-    }
+    // /// <summary>
```

Comment:
(Code maintenance) There is lot of commented out code. What are expected next steps with this code? Isn't it enough to just drop it?

Reply:
As I understand all the commented code will be removed in the future. Now it is commented to be possible to return easier previous state before separation. I can move out it of course, if it is a big problem.